### PR TITLE
KAA-1169 CC3200 documentation page

### PR DESCRIPTION
**Pull request by forGGe**

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 1]
```diff
@@ -5,4 +5,161 @@ permalink: /:path/
...
+then extract the archive, cd to it and execute the following for Linux:
+
+```
+mkdir -p build
+cd build
+cmake -DKAA_PLATFORM=cc32xx -DCMAKE_TOOLCHAIN_FILE=../toolchains/cc32xx.cmake ..
+make
+```
+
+For more details on building the C SDK for Linux, please refer to [this page]({{root_url}}/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/).
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md]: Avoid `this page`.

**Comment by rasendubi** [comment]
LGTM

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 2]
```diff
@@ -5,4 +5,162 @@ permalink: /:path/
...
+# Example
+
+To quickly start with the Kaa IoT platform, you can download one of the Kaa demo applications from the [Kaa Sandbox]({{root_url}}/Getting-started/) and run it on the CC3200 board. We recommend you to start with the ConfigurationDemo.
+
+Connect CC3200 LaunchPad to your PC through a micro-USB connector and execute the following in your terminal:
+
+```
+tar -xzf configuration_demo.tar.gz
+cd CConfigurationDemo
+./build.sh build
```
Comment (RostakaGmfun) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:147]: Please add a note that user should type `cc32xx` in the prompt.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 3]
```diff
@@ -5,4 +5,162 @@ permalink: /:path/
...
+/opt/kaa/gcc-arm-none-eabi/bin/arm-none-eabi-gdb -x /opt/kaa/cc3200-sdk/tools/gcc_scripts/gdbinit build/demo_client.afx
+```
+
+NOTE: If you want to see the debug output in the terminal, make sure to get connected to `/dev/ttyUSB{0,1}.`
+
+# Flashing (Windows only)
+
+Jumpers on the CC3200 board should be connected as shown below.
+
+![Jumpers](attach/jumpers_debug_mode_400.png)
```
Comment (RostakaGmfun) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:162]: I can't see the `attach/` folder with pin-out diagram. Have you added it to index?

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 4]
```diff
@@ -5,4 +5,162 @@ permalink: /:path/
...
+Refer to [C SDK Linux page]({{root_url}}/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/) for more details.
+
+## Building C SDK for Windows
+
+Open the Cygwin terminal and execute the following:
+
+```
+mkdir -p build
+cd build
+cmake.exe -G "Unix Makefiles" -DKAA_PLATFORM=cc32xx -DKAA_TOOLCHAIN_PATH=c:/cygwin/opt/kaa -DCMAKE_TOOLCHAIN_FILE=../toolchains/cc32xx.cmake ..
```
Comment (Dubland) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:134]: Use TOOLCHAIN_PATH instead KAA_TOOLCHAIN_PATH

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 5]
```diff
@@ -5,4 +5,162 @@ permalink: /:path/
...
+For more information, please refer to the official [CC3200-Getting_Started_Guide][cc3200-getting-started-guide]
+
+[cc3200-getting-started-guide]: http://www.ti.com/lit/ug/swru376d/swru376d.pdf
+
+
+# Creating applications based on C SDK
+
+## Building C SDK for Linux
+
+Before creating applications based on the C SDK, you should obtain the C SDK and build a static
```
Comment (Dubland) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:114]: This offer may be made as a general for Windows and Linux.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md (hunk 4 above)
Comment (forGGe) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:134]: Done

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md (hunk 3 above)
Comment (forGGe) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:162]: Done

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md (hunk 2 above)
Comment (forGGe) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:147]: Done

**Comment by forGGe** [comment]
Hotfixes are included in #782 and #784

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md (hunk 5 above)
Comment (forGGe) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:114]: Done

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 6]
```diff
@@ -5,4 +5,162 @@ permalink: /:path/
...
+            cd /etc/udev/rules.d/
+            sudo nano 98-usbftdi.rules
+    1. Write the following code into the added rule file and then save the file.
+
+            SUBSYSTEM=="usb", ATTRS{idVendor}=="0451", ATTRS{idProduct}=="c32a", MODE="0660", GROUP="dialout",
+            RUN+="/sbin/modprobe ftdi-sio" RUN+="/bin/sh -c '/bin/echo 0451 c32a > /sys/bus/usb-serial/drivers/ftdi_sio/new_id'"
+    1. Reload the rules.
+
+            sudo udevadm control --reload-rules
+    1. To use Openocd as a regular user, add yourself to the dialout group.
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:44]: OpenOCD

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 7]
```diff
@@ -5,4 +5,162 @@ permalink: /:path/
...
+
+            sudo usermod -a -G dialout <username>
+    1. Log out and log in to finish the process.
+    NOTE: The board should be enumerated as `/dev/ttyUSB{0,1}`. Use ttyUSB1 for UART.
+1. Install CC3200SDK.
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:49]: Sometimes _CC3200 SDK_ is written separately and sometimes together

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 8]
```diff
@@ -5,4 +5,162 @@ permalink: /:path/
...
+1. Install the GNU ARM toolchain: [gcc-arm-none-eabi](https://launchpad.net/gcc-arm-embedded)
+to the `opt\kaa` directory under the Cygwin root directory (default is `C:\cygwin`)
+1. Install [CC3200SDK](http://www.ti.com/tool/cc3200sdk) to the `opt\kaa`
+(if the directory doesn't exist, create it) directory under the Cygwin root directory.
+1. Install [cmake](http://www.cmake.org/) and add it bin directory to the system environment.
+
+To enable debugging for your CC3200 applications, you will also need to build OpenOCD as
+described in [the official CC3200-Getting_Started_Guide][cc3200-getting-started-guide] (item 3.3.3)
+
+For more information, please refer to the official [CC3200-Getting_Started_Guide][cc3200-getting-started-guide]
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:105]: Missed dot. 
In the previous sentence **the official**  is also related to the link.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 9]
```diff
@@ -5,4 +5,162 @@ permalink: /:path/
...
+[cc3200-getting-started-guide]: http://www.ti.com/lit/ug/swru376d/swru376d.pdf
+
+
+# Creating applications based on C SDK
+
+## Building C SDK for Linux
+
+Before creating applications based on the C SDK, you should obtain the C SDK and build a static
+library from it. To do so, generate the [C SDK in Admin UI]({{root_url}}/Administration-guide/Tenants-and-applications-management/#generating-endpoint-sdk),
+then extract the archive, cd to it and execute the following for Linux:
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:116]: Some changes were made here in another pull request. Currently we have: _Change directory to where SDK was unpacked and execute the following for:_
**for** is not needed.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 10]
```diff
@@ -5,4 +5,162 @@ permalink: /:path/
...
+* TOC
+{:toc}
+
+This guide explains how to build applications for Texas Instruments CC3200 LaunchPad
+(further, CC3200) based on the Kaa C SDK.
+
+# Installing necessary components for Linux (Ubuntu)
+
+Before building the C SDK for the CC3200 platform on Linux, you need to perform
+the following installation.
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:20]: _installation steps_ should be better.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md (hunk 6 above)
Comment (forGGe) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:44]: Done

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md (hunk 7 above)
Comment (forGGe) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:49]: Fixed

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md (hunk 8 above)
Comment (forGGe) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:105]: Fixed

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md (hunk 9 above)
Comment (forGGe) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:116]: Done

**Pull request merged**